projects/plugins/jetpack/extensions/blocks/donations/index.js

```diff
@@ -24,12 +25,44 @@ export const settings = {
 	description: __( 'Collect one-time, monthly, or annually recurring donations.', 'jetpack' ),
 	icon: DonationsIcon,
 	category: 'earn',
-	keywords: [ __( 'Donations', 'jetpack' ) ],
+	keywords: [ __( 'donations', 'jetpack' ) ],
+	attributes: {
+		[ ONE_TIME_DONATION ]: {
+			type: 'boolean',
+			default: true,
+		},
+		[ MONTHLY_DONATION ]: {
+			type: 'boolean',
+			default: true,
+		},
+		[ ANNUAL_DONATION ]: {
+			type: 'boolean',
+			default: true,
+		},
```

Comment:
I'm not sure if we should have constants as block attributes. 🤔

The risk is that folks might just accidentally change the value of one of these without fully realizing they are breaking the whole block.

Reply:
In that case, it would fail early. If you change something that would break the whole block, you would notice it instantly.